src/frequenz/sdk/microgrid/component_data.py

```diff
@@ -72,7 +89,24 @@ def from_proto(raw: microgrid_pb.ComponentData) -> MeterData:
 
 @dataclass(frozen=True)
 class BatteryData(ComponentData):
-    """A wrapper class for holding battery data."""
+    """A wrapper class for holding battery data.
+
+    soc: battery's overall SoC in percent (%).
+
+    soc_lower_bound: the soc lower bound of the current bounds of the metric, as
```

Comment:
I think the word `current bounds` in these descriptions can be confusing.

One option suitable for just SoC is "the SoC below which discharge command will be blocked by the system".  @tiyash-basu-frequenz does that make sense?

Reply:
I merged these two docstrings:
https://github.com/frequenz-floss/frequenz-api-microgrid/blob/v0.x.x/proto/frequenz/api/microgrid/common.proto#L60

https://github.com/frequenz-floss/frequenz-api-microgrid/blob/v0.x.x/proto/frequenz/api/microgrid/common.proto#L15

Reply:
Sure thing. You probably know better which attributes can be explained more precisely. Would you mind pointing these out and providing the correct documentation so that I can add it to the PR?

Reply:
Yup,  will do soon.

Reply:
I made a separate comment.